core/src/main/java/com/google/googlejavaformat/java/Formatter.java

```diff
@@ -135,6 +135,7 @@ static void format(
     DiagnosticCollector<JavaFileObject> diagnostics = new DiagnosticCollector<>();
     context.put(DiagnosticListener.class, diagnostics);
     Options.instance(context).put("allowStringFolding", "false");
+    Options.instance(context).put("-source", options.javaSourceLevel());
```

Comment:
Please add a TODO so I remember to remove this once the default language level is 9. Also you can use the `Option` enum instead of the string `"-source"`:

```java
// TODO(cushon): this should default to the latest supported source level, remove this after backing out
// https://github.com/google/error-prone-javac/commit/c97f34ddd2308302587ce2de6d0c984836ea5b9f
Options.instance(context).put(Option.SOURCE, "9");
```